### ISSUE TITLE
Enhancement: Red-Hat porting

### ISSUE THREAD

**Issue by red-erik** [healthchecks/healthchecks#831]
Hello,
thank you for your great software. Sadly, I'm not allowed to work on any distro other than Rhel (8/9) so, would it be possible to have a "quick" guide on how to have all dependencies installed ? I had it running way back and it was "complicated" and today, I'm quite lost trying to have all dependencies satisfied.
Thank you in advance for every suggestion.

Regards,
Red.

**Comment by red-erik** [comment]
![healtchecksio](https://github.com/healthchecks/healthchecks/assets/10245160/9a8fe3e0-b6cf-45e7-84be-8083560328c9)

**Comment by cuu508** [comment]
I think the SyntaxError in the screenshot is caused by an old Python version (Python 3.6 on RHEL 8?). The current version of Healthchecks works with Python 3.8 or later.

**Comment by red-erik** [comment]
True, thank you. I removed everything related to Python 3.6 or used pip3.8 to install packages, btw I still have

![572b7870-ad41-4a21-8055-dc2a2f43b13d](https://github.com/healthchecks/healthchecks/assets/10245160/e89dde8e-5c9a-40fb-a65f-6acae190144f)

**Comment by cuu508** [comment]
I'd like to try this myself. What RHEL version are you using, and what would be the the equivalent Fedora, CentOS or Rocky Linux version (I don't have a RHEL licence, and I assume I would need one)?

**Comment by red-erik** [comment]
I solved it rebuilding everything from scratch and using pip3.8 to install dependencies. Now I have to understand how to properly have both services running together (you already helped me ways back). At the end, I'll send you all steps I did to have it up"and"running on Rhel 8.
Regards,
Red.

**Comment by red-erik** [comment]
As promised...

sudo su -
yum -y install git
yum -y install gcc
yum -y install python38
yum -y install python38-devel
yum -y install libpq-devel
yum -y install openssl-devel
yum -y install libcurl 
yum -y install libcurl-devel

mkdir -p /webapps
cd /webapps

python3.8 -m venv hc-venv
source hc-venv/bin/activate

git clone https://github.com/healthchecks/healthchecks.git

pip3.8 install --upgrade pip
pip3.8 install -r healthchecks/requirements.txt
pip3.8 install uwsgi

cd  /webapps/healthchecks

./manage.py migrate
./manage.py createsuperuser
./manage.py test
./manage.py runserver 0.0.0.0/8000

firewall-cmd --add-port=8000/tcp

Still to properly manage both services... ;-)

Regards,
Red.
